src/_locales/en/messages.json

```diff
@@ -48,16 +48,37 @@
   "keyRevoked_action_reconnectBtn": {
     "message": "Reconnect"
   },
+  "pay_action_pay": {
+    "message": "Send now"
+  },
   "pay_error_notEnoughFunds": {
     "message": "Insufficient funds to complete the payment."
   },
+  "pay_error_outgoingPaymentFailed": {
+    "message": "We were unable to process this transaction. Please try again!",
+    "description": "OutgoingPayment.failed === true"
```

Comment:
Just to be clear, `OutgoingPayment.failed = true` and `OutgoingPayment.sent_amount > 0` is possible?

We want to treat any partial amount to be treated as success here?

Reply:
> Just to be clear, OutgoingPayment.failed = true and OutgoingPayment.sent_amount > 0 is possible?
Yes.

> We want to treat any partial amount to be treated as success here?
We cannot treat a sent partial amount as an error (`failed = true && sent_amount = 0`), since an amount of was actually sent. We can use `payStatus.type = 'partial'` for this.

Reply:
Discussed cases with @raducristianpopa on Slack. We need to handle these cases:
```
failed = T; sent_amount = 0             => definitely error. immediate fail.
failed = F; sent_amount = 0             => we don't know yet. maybe warn user? It could be an error if check for long enough, or sent_amount might become non-zero

failed = T; sent_amount > 0             => ok, but only a partial was sent
failed = F; sent_amount > 0             => ok, but only a partial was sent, can be successfully completed afterwards

failed = T; sent_amount = debit_amount  => not possible
failed = F; sent_amount = debit_amount  => ok. immediate success.
```

@RabebOthmani How to best show this to user? For added context, after the payment, we've to repeatedly check payment status (polling). Right now, we check for up to 8 seconds only. The payment might be in any of above 5 states.

Also, note that, we'll be showing user a spinner for those 8s. Might be better to show "verifying payment status" message while we are in verify stage?

Reply:
8s is a long time for just a spinner. My unsolicited opinion is that we should show some kind of messaging for sure.

Reply:
Per Slack: 
* " It looks like this is taking long. Check your wallet later to ensure the payment went through" 
* Partial payment- only <amount> was successfully sent. to learn more <link>